cmd/restic/global.go

```diff
@@ -101,6 +102,7 @@ func init() {
 
 	f := cmdRoot.PersistentFlags()
 	f.StringVarP(&globalOptions.Repo, "repo", "r", os.Getenv("RESTIC_REPOSITORY"), "`repository` to backup to or restore from (default: $RESTIC_REPOSITORY)")
+	f.StringVarP(&globalOptions.RepositoryFile, "repository-file", "", os.Getenv("RESTIC_REPOSITORY_FILE"), "read the repository from a `file` (default: $RESTIC_REPOSITORY_FILE)")
```

Comment:
Please change `read the repository from` to `read the repository location from`, as without that it can sound like the repository is stored in the file. The wording "repository location" is used in the documentation at https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html .